docs/Supported-Data-Types.md

```diff
@@ -0,0 +1,42 @@
+# Supported Data Types
+
+cBioPortal is a multimodal cancer data visualization tool and supports a variety of data types. For some data types, we have explicit support; for others, you can leverage generic assay or clinical data to assign arbitrary data to either a sample or patient. Beyond including data directly into cBioPortal, one can also link in additional external viewers specialized for particular data types, or even entirely different dashboards or portals. This combination of structured and flexible data formats, as well as the option to link out to other viewers, allows cBioPortal to be used for many different combinations of clinical data and profiling methods.
+
+## Molecular Data
+
+| Assay | Processed Data | cBioPortal Data Type | Example cBioPortal Study |
+| --- | --- | --- | --- |
+| Bulk DNA Sequencing | Mutations | [Mutations](./File-Formats.md#mutation-data) | [MSK-IMPACT (Nat Med, 2017)](https://www.cbioportal.org/study/summary?id=msk_impact_2017) |
+| - | Copy Number Alterations per Gene (CNA) | [CNA](./File-Formats.md#discrete-copy-number-data) | [MSK-IMPACT (Nat Med, 2017)](https://www.cbioportal.org/study/summary?id=msk_impact_2017) |
+| - | Arm Level CNA Data | [Generic Assay](./File-Formats.md#arm-level-cna-data) | [TCGA Pancan (Cell 2018)](https://www.cbioportal.org/study/summary?id=acc_tcga_pan_can_atlas_2018) |
+| - | Copy Number Segmentation | [Segmented Data](./File-Formats.md#segmented-data) | [MSK-IMPACT (Nat Med, 2017)](https://www.cbioportal.org/patient?studyId=msk_impact_2017&caseId=P-0000004) |
+| - | Structural Variants | [Structural Variants](./File-Formats.md#structural-variant-data) | [PCAWG (Nature, 2020)](https://www.cbioportal.org/study/summary?id=pancan_pcawg_2020) |
+| - | Mutational Signatures | [Generic Assay](./File-Formats.md#mutational-signature-data) | [PCAWG (Nature, 2020)](https://www.cbioportal.org/study/summary?id=pancan_pcawg_2020) |
+| - | Genetic Ancestry | [Generic Assay](./File-Formats.md#generic-assay) | [TCGA Pancan (Cell, 2018)](https://www.cbioportal.org/study/summary?id=acc_tcga_pan_can_atlas_2018) |
+| Bulk RNA Sequencing | Gene Expression | [Gene Expression](./File-Formats.md#expression-data) | [TCGA Pancan (Cell, 2018)](https://www.cbioportal.org/study/summary?id=acc_tcga_pan_can_atlas_2018) |
```

Comment:
Expression?

Reply:
Do you mean RNA Expression - rather than Gene expression? E.g.:

```suggestion
| Bulk RNA Sequencing | Gene Expression | [RNA Expression](./File-Formats.md#expression-data) | [TCGA Pancan (Cell, 2018)](https://www.cbioportal.org/study/summary?id=acc_tcga_pan_can_atlas_2018) |
```